CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 33 Release Notes
 
+### Query endpoint
+
+#### Query workflows by Submission time
+
+Cromwell now supports querying workflows by submission time. This will help find workflows that are submitted but not started yet (i.e. workflows which are
+in On Hold state). Similar to the start and end timestamps, you can include `submission:<timestamp>` to fetch workflows who have submission time greater or equal to specified timestamp.
```

Comment:
ToL: to keep these short, I would stop at `On Hold state).` and then link directly to the docs for the rest.